plugins/catalog-backend/config.d.ts

```diff
@@ -182,5 +181,20 @@ export interface Config {
      * housing catalog-info files.
      */
     processingInterval?: HumanDuration;
```

Comment:
Maybe silly, but should setting the interval to 0 disable processing instead?

Reply:
Hmm, it's interesting. But it would be `processingInterval: { seconds: 0 }` which is a little odd. Maybe `processingInterval?: HumanDuration | boolean`?

Reply:
Or even `HumanDuration | false` specifically?